Mage.Sets/src/mage/cards/c/CaseOfThePilferedProof.java

```diff
@@ -0,0 +1,175 @@
+package mage.cards.c;
+
+import java.util.Map;
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.CaseAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.PermanentsOnTheBattlefieldCondition;
+import mage.abilities.condition.common.SolvedSourceCondition;
+import mage.abilities.decorator.ConditionalReplacementEffect;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.ReplacementEffectImpl;
+import mage.abilities.effects.common.counter.AddCountersTargetEffect;
+import mage.abilities.hint.common.CaseSolvedHint;
+import mage.constants.ComparisonType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.events.CreateTokenEvent;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.ClueArtifactToken;
+import mage.game.permanent.token.Token;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author DominionSpy
+ */
+public final class CaseOfThePilferedProof extends CardImpl {
+
+    public CaseOfThePilferedProof(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{W}");
+
+        this.subtype.add(SubType.CASE);
+
+        // Whenever a Detective enters the battlefield under your control and whenever a Detective you control is turned face up, put a +1/+1 counter on it.
+        Ability initialAbility = new CaseOfThePilferedProofTriggeredAbility();
```

Comment:
is this affected by same bug as #11719 ?

Reply:
Yes, it looks like it is

Reply:
this is good to go once that gets fixed separately

Reply:
Should be good now.
Could you rebase and check the card is working on top of master?

Reply:
Yes that works now, thank you